leap.hs

```diff
@@ -1,10 +1,11 @@
 --http://exercism.io/exercises/haskell/leap/readme
 checkLeap yr = if (yr `mod` 4) == 0
     then if (yr `mod` 100) /= 0
-            then putStrLn "Leap"
+            then True
         else if (yr `mod` 400) /= 0
-            then putStrLn "not Leap"
-            else putStrLn "Leap"
-        else putStrLn "not Leap"
+            then False
+            else True
+        else False
+main :: IO()
 main = do
```

Comment:
Please read the first note at https://www.vacationlabs.com/haskell/basic-types-and-functions.html#hello-world